### PR TITLE
Migrated from bower to yarn

### PR DESCRIPTION
**Pull request by tanmayanand44**
I have migrated the current web package manager i.e. bower to yarn. Issue(#253)

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "test": "./node_modules/.bin/karma start --single-run --browsers Chrome"
   },
   "dependencies": {
-    "jasmine-core": "^2.5.2"
+    "jasmine-core": "^2.5.2",
+    "package.json": "^2.0.1"
```
Comment (agentmilindu) [inline ui/package.json]: Hi @tanmayanand44, What this line is about? Can you explain what you did in this PR?

Comment (tanmayanand44) [reply ui/package.json]: I ran "yarn add package.json" to  install all the dependencies  to package.json.

Comment (agentmilindu) [reply ui/package.json]: Oh, no, That's not how it is done. Did you go through the migration guide given in the warning message in the issue? https://bower.io/blog/2017/how-to-migrate-away-from-bower/

Comment (tanmayanand44) [reply ui/package.json]: Yes,read it.But got very little how to implement.
Can u guide me a little.

Comment (agentmilindu) [reply ui/package.json]: Yeah, sure! 

First, install Yarn on your machine. Then navigate to the `ui` folder of Bassa and issue the following commands.
```
yarn global add bower-away
bower-away
```

Then run `bower-away --apply` and then `rm -rf bower_components`. Finally, run `yarn`. 

You might get some errors while Yarn is installing the dependencies. That's okay, commit your changes, push to your repo and create a PR. Comment the issues you got while Yarn install.  I'll tell you how to fix them then are there.

But start with a clean branch checkout from `master` ( `git checkout -b 253-bower-migration master ).  Don't include irrelevant commits in the PR like `Tanmay Anand Contribution`.

Comment (tanmayanand44) [reply ui/package.json]: when executing bower-away command it shows  'command not found'.

Comment (tanmayanand44) [reply ui/package.json]: All the commands worked except bower-away and bower-away --apply

Comment (agentmilindu) [reply ui/package.json]: You ran `yarn global add bower-away` right? Did it install correctly?

Comment (tanmayanand44) [reply ui/package.json]: Yes it it did.

Comment (agentmilindu) [reply ui/package.json]: Hmm, maybe give it a try again after restarting the machine?

**Comment by rehrumesh** [comment]
@tanmayanand44 Please update the master branch.

**Comment by tanmayanand44** [comment]
@rehrumesh Sir,I didn't understand.Can u a bit more clear what I need to do.

**Comment by agentmilindu** [comment]
Relates to #305

**Pull request closed without merging**